docs/advanced/testing.md

```diff
@@ -243,7 +243,7 @@ it('should update the list', done => {
     </Provider>
   );
 
-  setTimeout(() => {
+  setInterval(() => {
```

Comment:
I don't feel comfortable doing `setInterval` in our tests without showing a proper cleanup 😅 we could change this test to instead assert that after 200ms we have the old length of children + an extra one due to the subscription firing.

Reply:
I see what you mean. If you still wanted to show testing the interval. Would there be anything wrong when testing with the cleanup like below?

`
 const interval = setInterval(() => {
    expect(wrapper.find('.list').children()).toHaveLength(index + 1); // See how many items are in the list
    index++;
    if (index === 2) {
      clearInterval(interval);
      done();
    }
  }, 200);
`

Reply:
Well if the test fails you risk never calling `clearInterval`

Reply:
Oops. How about adding a before and after hook in the snippet like this?

```
  before(function() { let interval; });
  after(function() { clearInterval(interval) });
```

If doing "setTimeout", the snippet would get shorter doing something like this I think

```
  setTimeout(() => {
    expect(wrapper.find('.list').children()).toHaveLength(index + 1); // See how many items are in the list
    done();
  }, 200);
```